Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/util/OneSignalPrefs.java

```diff
@@ -0,0 +1,70 @@
+package com.onesignal.sdktest.util;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.onesignal.sdktest.constant.Text;
+
+public class OneSignalPrefs {
+
+    private static final String APP_SHARED_PREFS = "com.onesignal.sdktest";
+
+    public static final  String OS_APP_ID_SHARED_PREF = "OS_APP_ID_SHARED_PREF";
+    private static final String PRIVACY_CONSENT_SHARED_PREF = "PRIVACY_CONSENT_SHARED_PREF";
+    public static final String USER_EMAIL_SHARED_PREF = "USER_EMAIL_SHARED_PREF";
+    public static final String USER_EXTERNAL_USER_ID_SHARED_PREF = "USER_EXTERNAL_USER_ID_SHARED_PREF";
+    private static final String LOCATION_SHARED_PREF = "LOCATION_SHARED_PREF";
+
+    private static SharedPreferences getSharedPreference(Context context) {
+        return context.getSharedPreferences(APP_SHARED_PREFS, Context.MODE_PRIVATE);
+    }
+
+    public static boolean exists(Context context, String key) {
+        return getSharedPreference(context).contains(key);
+    }
+
+    public static String getOneSignalAppId(Context context) {
+        return getSharedPreference(context).getString(OS_APP_ID_SHARED_PREF, "77e32082-ea27-42e3-a898-c72e141824ef");
```

Comment:
OneSignal app id should be static called something like `ONESIGNAL_EXAMPLE_APP_ID` instead of a string here.

Reply:
Got it